x/stakeibc/keeper/hooks.go

```diff
@@ -290,22 +297,24 @@ func (k Keeper) TransferExistingDepositsToHostZones(ctx sdk.Context, epochNumber
 			k.Logger(ctx).Info("TransferExistingDepositsToHostZones msg:", msg)
 			_, err := k.TransferKeeper.Transfer(goCtx, msg)
 			if err != nil {
-				k.Logger(ctx).Error("\t[TRANSFER] ERROR WITH DEPOSIT RECEIPT", hostZone.TransferChannelId, transferCoin, addr, delegateAddress, timeoutHeight)
-				pstr = fmt.Sprintf("\t[TRANSFER] ERROR WITH DEPOSIT RECEIPT {%v}", err)
+				k.Logger(ctx).Error("\t[TransferExistingDepositsToHostZones] ERROR WITH DEPOSIT RECEIPT", hostZone.TransferChannelId, transferCoin, addr, delegateAddress, timeoutHeight)
```

Comment:
same, you might want to use `fmt.Sprintf` here

Reply:
do you have to use %s, %d, etc?

Reply:
Done.